### PR TITLE
fix README's

### PR DESCRIPTION
**Pull request by fjmolinas**
This PR fixes some README typos and missing instructions.

Only one example solution has been improved  (riot-basics/rtc)

### CODE REVIEW

File: riot-basics/gpio/README.md [hunk 1]
```diff
@@ -16,6 +16,19 @@ used to toggle an on-board LED.
 
 **Objective:**
 
+- Add the GPIO's feature to the requirements of the application (in the `Makefile`):
```
Comment (aabadie) [inline riot-basics/gpio/README.md:17]: You could also remove the line "Objective" above. I think this is a leftover from previous versions.

File: riot-basics/rtc/.solution/main.c [hunk 2]
```diff
@@ -45,7 +45,11 @@ static void *blink_thread(void *arg)
         rtc_get_time(&time);
         printf("Current time: %d:%d:%d\n",
                time.tm_hour, time.tm_min, time.tm_sec);
-        time.tm_sec += 5;
+        time.tm_sec  += 5;
```
Comment (aabadie) [inline riot-basics/rtc/.solution/main.c]: This change should not be part of this PR, so just remove the corresponding commit. I'll push another (better) fix later.

File: riot-networking/sock-udp/README.md [hunk 3]
```diff
@@ -89,17 +89,21 @@ Both applications will run as RIOT native instances.
...
+    ```
   - Implement the listening of the echo sent by the server:
     ```c
-    sock_udp_ep_t remote;
     ssize_t res;
     if ((res = sock_udp_recv(&sock, buf, sizeof(buf), SOCK_NO_TIMEOUT,
                              &remote)) >= 0) {
 
         printf("Message received: %s\n", (char*)buf);
-        if (sock_udp_send(&sock, buf, res, &remote) < 0) {
```
Comment (aabadie) [inline riot-networking/sock-udp/README.md:100]: If you remove this, the client won't receive the message echoed from the server.

File: riot-basics/rtc/README.md [hunk 4]
```diff
@@ -21,5 +22,8 @@ In this exercise, we propose to experiment the RTC API.
 
 6. In the RTC alarm callback, send a message to the `blink_thread
 
+_Note: watch out, the rtc_set_alarm() won't work if elements in `struct tm *time`,
```
Comment (aabadie) [inline riot-basics/rtc/README.md]: This note won't be needed after I push my alarm time computation fix.

**Review by aabadie — CHANGES_REQUESTED** [review]
Thanks for fixing the exercices readmes. I have some remarks though. Can you address them?

File: riot-networking/sock-udp/README.md (hunk 3 above)
Comment (fjmolinas) [reply riot-networking/sock-udp/README.md:100]: That is correct but that is code belonging on the server side, not client.

**Comment by fjmolinas** [comment]
@aabadie changes addressed

**Review by aabadie — APPROVED** [review]

**Pull request merged**